src/debug/sdk_path.ts

```diff
@@ -0,0 +1,30 @@
+"use strict"
+
+// This file exists to write the SDK path from config in the extension and read it from the
+// debug adapter. Everything imported here must be available in both places (eg. no vscode).
+import * as os from "os";
+import * as path from "path";
+import * as fs from "fs";
+
+const sdkPathStoreName = "dart-code.sdk";
+const sdkPathStore = path.join(os.tmpdir(), sdkPathStoreName);
```

Comment:
`tempdir()`, or somewhere in user home (`~/.dartcode/.settings`)? I have no preference but wanted to call out the option.